contributing/testing-guide.md

```diff
@@ -161,4 +208,40 @@ Chromatic is set-up to check for regressions in all component stories. Typically
           └── [component-name]-test.stories.* (private stories)
 ```
 
-To introduce a new visual test, create a story in the relevant stories file that demonstrates the behaviour and it will automatically be detected by Chromatic.
+To introduce a new visual test, create a story in the relevant stories file that demonstrates the behaviour.
```

Comment:
I really like this amendment with the additional information. It's well explained, with examples. Nice!